BrayanSantos/question-01/index.js

```diff
@@ -0,0 +1,17 @@
+const pokemons = require('./pokeDB.json')
+
+function search(id) {
+    for (var pokemon of pokemons) {
+        if (pokemon.id == id) {
+            console.log("name: " + pokemon.name);
+            console.log("type: " + pokemon.type);
+            return true;
+        }
+    }
+    return false;
+}
+
+let encontrado = search(process.argv[2]);
```

Comment:
Next time try to use this approach: https://github.com/univas/2017-si6-firstexam/blob/368623538c7472ecc3169bbccabbecf4fd9fc1af/correction/question-03/check-students.js#L28